interfaces/HO-Portal/src/app/models/program-metrics.model.ts

```diff
@@ -5,8 +5,9 @@ export class ProgramMetrics {
 }
 
 export class PeopleMetrics {
-  pendingVerification: number;
-  verifiedAwaitingDecision: number;
+  startedEnlisting: number;
```

Comment:
Why "Enlisting" as the verb?

We use "Enroll" in all kinds of other places?

Reply:
I did that because we have a column in the back-end that is called programsEnrolled which actually contains the programs for which you are verified. I therefore i wanted to use a different term for this field to avoid confusion.

Reply:
Maybe we can use register instead of enlist? That also feels like it fit's more into the conext of internet accounts.